Makefile

```diff
@@ -1,6 +1,25 @@
 GOBASE=$(shell pwd)
 GOBIN=$(GOBASE)/bin
 GO_BUILD_FLAGS :=-tags 'containers_image_openpgp osusergo exclude_graphdriver_btrfs exclude_graphdriver_devicemapper'
+ROOT_DIR := $(or ${ROOT_DIR},$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST)))))
+
+VERBOSE ?= false
+REPORTS ?= $(ROOT_DIR)/reports
+
+GO_TEST_FORMAT = pkgname
+ifeq ($(VERBOSE), true)
+        GO_TEST_FORMAT=standard-verbose
+endif
+
+GINKGO_REPORTFILE := $(or $(GINKGO_REPORTFILE), ./junit_unit_test.xml)
+GO_UNITTEST_FLAGS = --format=$(GO_TEST_FORMAT) $(GOTEST_PUBLISH_FLAGS) -- -count=1 -cover
+GINKGO_UNITTEST_FLAGS = -ginkgo.focus="$(FOCUS)" -ginkgo.v -ginkgo.skip="$(SKIP)" -ginkgo.v -ginkgo.reportFile=$(GINKGO_REPORTFILE)
+
+PSQL_IMAGE := $(or ${PSQL_IMAGE},quay.io/centos7/postgresql-12-centos7:latest)
```

Comment:
is there an issue using in memory DB?

Reply:
For testing the data store subsystem, it IMHO makes sense to use the DB we target for production, so we don't miss corner cases where GORM's drivers have slightly different behaviors.

Otherwise, would sqlite be lightweight enough for testing? Or if it's about performance, use Redis  as cache layer to to accelerate reads?

Reply:
for subsystem (integration) tests i agree but for unit tests in memory should be enough IMO. 
gorm support sqllite https://gorm.io/docs/connecting_to_the_database.html#SQLite